### ISSUE TITLE
synclounge 2.0 beta player buffers forever

### ISSUE THREAD

**Issue by tomh4x** [synclounge/synclounge#107]
When loading video, sometimes the 2.0 beta player will show the circle indefinitely.  Strangely enough, this is often alleviated by the host clicking the settings "cog" and manually setting the audio track.  This is true even when the video container only has a single audio track.

This issue is occurring for multiple users (regardless of who hosts or what browser or OS) though I've personally only used chrome (latest stable) on fully updated windows 10.

**Comment by TylerBoire** [comment]
I can second this issue, Windows 10 Firefox & Chrome, as well as Kali/Ubuntu Firefox. Multiple users, regardless of who hosts the room.

**Comment by samcm** [comment]
Thanks for reporting the issue 👍. I'm just about to leave to travel for work but I can investigate when I'm back (roughly 1 week). 

Does it happen with every file or just one? Which version of Plex Media Server?

**Comment by tomh4x** [comment]
the occurrence seems entirely arbitrary, but yes it has happened across multiple files/types of video containers.

**Comment by tomh4x** [comment]
.

**Comment by hbh7** [comment]
I keep seeing this pop up. Seems to be more common to guests but I've seen it as the host too. Will try out the audio track change fix

**Comment by aptalca** [comment]
I noticed, in many cases, if I do the resume option for the synclounge player, I get the forever load/buffer, but if I click on the player bar to seek, it starts playing from the new location immediately

**Comment by hbh7** [comment]
Wait no it's definitely still a thing

**Comment by ttshivers** [comment]
This issue will very likely be resolved in an upcoming release as we are completely swapping out our video player.

**Comment by tomh4x** [comment]
But its NOT resolved.  @samcm replied saying he would address it then ghosted me for a year despite numerous messages on the project's discord as well.

**Comment by samcm** [comment]
I’ll reopen, it was closed by the bot for inactivity. @ttshivers is spot on, I imagine it’ll be fixed by that massive and impressive piece of work 👌.

@tomh4x I don’t even know where to start with your comment. Everyone here works on SL in their spare time. Hundreds of hours & dollars have been spent just to make life a little easier when watching Plex with friends and family. That’s fine, it’s a service we enjoy providing, but I won’t accept disrespect . If you want to fix it, fix it.

**Comment by tomh4x** [comment]
You said you'd fix it over a year ago and its your project...  You ignored (as in flat out never acknowledged my existence WHILE chatting). I also work more than full time.  You had over a year and didn't communicate any delay or answer me when I offered to fix it.

I really don't want to hear it.  You earned the disrespect.  Jackass.

**Comment by MagicalCodeMonkey** [comment]
@tomh4x You may want to review the [GitHub Community Guidelines](https://help.github.com/en/github/site-policy/github-community-guidelines) as you have violated them.

**Comment by hbh7** [comment]
Still seeing the issue sometimes. I think we're waiting on the new update though.

**Comment by ttshivers** [comment]
In the mean time, check out https://github.com/ttshivers/synclounge

**Comment by 13steinj** [comment]
@ttshivers (sorry if this is not the place) but are you aware if your work / updated version also fixes unnecessary transcodes on optimized versions? I've found for some reason that on this root repo, even when playing an already optimized version Plex decides to transcode. Yet when playing via the Plex Web Player, it works fine.

If not / you don't know what I'm referring to, should I post a separate issue?

**Comment by ttshivers** [comment]
Yeah, the fork I have made should not have those unnecessary transcodes. It uses a different player (the same as what plex web uses), so direct stream and direct play support are improved.

**Comment by 13steinj** [comment]
Great to hear! Thanks @ttshivers , I'll check it out.

**Comment by jiggyjigsj** [comment]
@ttshivers Seems like I am running similar issue while trying to use the self hosted version.  I noticed that there was a release done for both your fork as well as this repo.

Tracking down on my side, it seems to error out trying to query my Plex. Plex seems to return a 404 for a few of the calls. 

Let me know what else I can provide. I know your fork use to work a few weeks back. For the time being, I'll attempt to find the last released it work for me. 

Error i see in Chrome: 
![image](https://user-images.githubusercontent.com/16803513/95706086-803c0700-0c1b-11eb-8300-e045639ed546.png)

URL it tries to query: 
* https://server.domain.com/video/:/transcode/universal/session/SESSION_ID/base/00000.ts
